assets/app/view/game/round/operating.rb

```diff
@@ -86,6 +86,10 @@ def render
           elsif (company = entity).company?
             left << h(Company, company: company)
 
+            # render combos if blocking for a private company with
+            # must_lay_together=true
+            left << h(Game::Abilities, user: @user, game: @game, combos_only: true)
```

Comment:
Instead of passing `combos_only` here, should I just create a new view for the `Combine with` section?